sql/aggregate/block.sql

```diff
@@ -1,8 +1,8 @@
 DROP TABLE IF EXISTS AGGREGATE_block_{{ decade }};
 SELECT 
     boro,
-    bctcb{{ decade }},
-    cenblock{{ decade }},
```

Comment:
This line introduces a duplicate column in the aggregate census tract output as discussed @td928